src/Microsoft.ML.DnnAnalyzer/Microsoft.ML.DnnAnalyzer/Microsoft.ML.DnnAnalyzer.csproj

```diff
@@ -4,7 +4,6 @@
     <OutputType>Exe</OutputType>
     <TargetFramework>netcoreapp2.0</TargetFramework>
     <AssemblyName>DnnAnalyzer</AssemblyName>
-    <IncludeInPackage>Microsoft.ML.TensorFlow</IncludeInPackage>
   </PropertyGroup>
 
   <ItemGroup>
```

Comment:
Existing Issue: if you want this project to work on Linux/Mac you also need tensorflow_framework below, similar to https://github.com/dotnet/machinelearning/blob/eb264892e75219b1f191d3c35fcbc636186b8acf/test/Microsoft.ML.Tests/Microsoft.ML.Tests.csproj#L33.

Reply:
@yaeldekel I think this needs to be updated.

Reply:
I'm not sure fixing this issue is 100% necessary for `0.6` since we aren't going to be shipping DnnAnalyzer in `0.6`.